src/collections/_documentation/development/server/environment.md

```diff
@@ -1,57 +1,38 @@
 ---
 title: 'Development Environment'
-sidebar_order: 1
+sidebar_order: 2
 ---
 
-## Setting up Your Development Environment
+This guide will step you through configuring a local development environment for the Sentry server on macOS. If you're using another operating system the instructions are still roughly the same, but we don't maintain any official documention.
 
-### Linux
+#### A quick note on Bash vs Zsh...
 
-Follow [the Docker file in the repository](https://github.com/getsentry/sentry/blob/master/docker/Dockerfile) but instead of creating and installing a wheel, install the package in editable mode (`pip install . -e`).
+macOS Catalina will recommend that you use `zsh` instead of `bash`. While the differences are minimal, this guide primarily covers bash. You can verify your shell by typing `echo $SHELL`.
 
-### macOS
+## Clone the Repository
 
-{% capture __alert_content -%}
-macOS Catalina will recommend that you use the zsh shell, instead of bash. While the differences are minimal, this guide only covers bash, so for best results please use bash. You can verify this by typing `echo $SHELL`, which should result in `/bin/bash`.
-
-{%- endcapture -%}{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-  level="warning"
-%}
-
-To get started, fork the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry). Clone your forked repo and go into it:
+To get started, clone the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry) - or, your fork if you are not a member of `@getsentry`.
 
 ```bash
-git clone https://github.com/<your github username>/sentry.git
+git clone https://github.com/getsentry/sentry.git
 cd sentry
 ```
 
-{% capture __alert_content -%}
-It's important that you remain in the repo's root directory for the rest of the setup! Also, please do the following steps in the exact order they're shown.
-{%- endcapture -%}{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-  level="warning"
-%}
-
+You're going to be working out of the repository for the remainder of the setup.
 
-### macOS - System Dependencies
+## System Dependencies
 
 Install [Homebrew](http://brew.sh), and then run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`.
 
 One thing that requires manual attention is `docker`, which should have just been installed. Open up Spotlight, search for "Docker" and start it. You should soon see the docker icon in your macOS toolbar. Docker will automatically run on system restarts, so this should be the only time you do this.
 
 You can verify that docker is running by running `docker ps`. If it doesn't error with something like `Error response from daemon: dial unix docker.raw.sock: connect: connection refused`, you're good to continue.
```

Comment:
If a customer does see that error, or another that is similar, should they pause, fix, then try again?

Reply:
i dont know, but this isnt customer facing docs, its for our developers + external contributors

there's a moderate expectation of "you get paid to do this for a living, figure it out" in these kinds of scenarios